rails/app/controllers/portal/clazzes_controller.rb

```diff
@@ -38,7 +38,7 @@ def index
     # PUNDIT_REVIEW_SCOPE
     # PUNDIT_CHECK_SCOPE (did not find instance)
     # @clazzes = policy_scope(Portal::Clazz)
-    @portal_clazzes = Portal::Clazz.all
+    @portal_clazzes = Portal::Clazz.search(params[:search], params[:page], nil)
```

Comment:
This seems like a big change - I'm surprised this search feature was missing.

Reply:
Ah, I see this is due to adding the `shared/collection_menu` to the template.

Reply:
There wasn't a page actually listing classes before, as far as I could see. So search was unnecessary.